### PR TITLE
Stories Component 

### PR DESCRIPTION
**Pull request by hugomarquesdev**
#263

### CODE REVIEW

File: utils/countries.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    const phoneNumber = parsePhoneNumber(pnumber);
+
+    if (phoneNumber?.country) {
+        const { emoji, name } = countries[phoneNumber.country];
+
+        return `${emoji} ${name}`;
+    }
+
+    return 'Unknown';
+}
```
Comment (obernardovieira) [inline utils/countries.tsx:29]: You won't need this, neither the import `import parsePhoneNumber from 'libphonenumber-js';` Please remove.

File: utils/countries.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    [key: string]: {
+        name: string;
+        native: string;
+        currency: string;
+        languages: string[];
+        emoji: string;
+    };
+} = countriesJSON;
+
+export const countriesOptions = Object.entries(countries).map(([key, value]: any) => ({
```
Comment (obernardovieira) [inline utils/countries.tsx:14]: seems like you also don't use this

File: utils/countries.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+export function getCountryNameFromInitials(countryInitials: string) {
+    if (countries[countryInitials]) {
+        return countries[countryInitials].name;
+    }
+
+    return 'Unknown';
+}
+
+export function getCountryCurrency(country: string) {
```
Comment (obernardovieira) [inline utils/countries.tsx:39]: seems like you also don't use this

**Review by obernardovieira — APPROVED** [review]

File: src/components/Stories/Stories.tsx [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+        const getStories = async () => {
+            const stories = await Api.getStories();
+
+            setStories(stories);
+        };
+
+        getStories();
+    }, []);
+
+    console.log('Prismic Data: ', storiesPrismicData);
```
Comment (PauloSousapt) [inline src/components/Stories/Stories.tsx:87]: Can we remove these logs?

**Review by PauloSousapt — APPROVED** [review]

**Pull request merged**